api/pwdb.proto

```diff
@@ -305,7 +305,7 @@ message ChallengeValidation {
   string author_comment = 101;
   string corrector_comment = 102;
   string passphrase_key = 103;
-  string passphrase = 104;
+  repeated string passphrases = 104;
```

Comment:
it won't works in db, for now I suggest to just store it in json in a `string` format

Reply:
Okay no prob